tests/end2end/test_hints_html.py

```diff
@@ -106,7 +106,11 @@ def test_hints(test_name, zoom_text_only, zoom_level, find_implementation,
         quteproc.set_setting('hints.find_implementation', find_implementation)
     quteproc.send_cmd(':zoom {}'.format(zoom_level))
     # follow hint
-    quteproc.send_cmd(':hint links normal')
+    if 'button' in test_name:
+        # We are hinting buttons, link hinting will not work
+        quteproc.send_cmd(':hint all normal')
+    else:
+        quteproc.send_cmd(':hint links normal')
```

Comment:
I think it's fine to just do `all` here unconditionally (also see https://github.com/qutebrowser/qutebrowser/pull/3474/files#diff-018770f170933d6da6d548ecf97c48fe).